reservations/signals.py

```diff
@@ -0,0 +1,20 @@
+from django.conf import settings
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from notifications.enums import NotificationType
+from notifications.utils import send_notification
+from .models import Reservation
+
+
+@receiver(post_save, sender=Reservation)
+def reservation_notification_handler(sender, **kwargs):
```

Comment:
IMO would be cleaner to do like `def reservation_notification_handler(sender, instance, created, **kwargs):`